recipe/meta.yaml

```diff
@@ -33,19 +33,19 @@ requirements:
         - xerces-c
         - libnetcdf
         - krb5
-        - libtiff
         - libpng
         - openjpeg
+        - libtiff 4*
         - curl
         - expat
+        - jpeg 9*
         - zlib  # [not win]
-        - jpeg  # [not win]
         - postgresql  # [linux]
         - jasper  # [not win]
         - giflib  # [not win]
         - json-c  # [not win]
         - freexl  # [not win]
-        - xz  # [not win]
+        - xz 5.2*  # [not win]
```

Comment:
@pelson I should have all be pinning from #39 and #38 here.

Note that `xz 5.2*` will fail on `Python 3.5`. Local tests were OK for `Python 2.7` and `3.4` (Linux), let's see how OS X and Windows behave now.